lib/themes/dosomething/paraneue_dosomething/js/campaign/OrganDonation.js

```diff
@@ -16,7 +16,9 @@ class OrganDonation extends Takeover {
 
     this.postalCodeUrl       = `${this.baseUrl}/organize/postal-code`;
     this.registrationUrl     = `${this.baseUrl}/organize/registration`;
+    this.shareUrl            = `${this.baseUrl}/organ-donation/registration`;
```

Comment:
I feel like `shareUrl` is a little confusing here since it feels like that would relate to social sharing...maybe like `storeRegistrationUrl` or something more explicit?